docs/conf.py

```diff
@@ -50,3 +56,19 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# -- Custom tooling ----------------------------------------------------------
+import os
+import subprocess
+
+def setup(app):
+    app.connect('builder-inited', build_changelog)
+
+def build_changelog(app):
+    docs_dir = os.getcwd()
+    os.chdir('../')
+    cmd = ['bash', os.getcwd() + '/tooling/build_changelog', os.getcwd()]
```

Comment:
I would use an f-string for the middle argument.
It may also be worth pulling the `os.getcwd()` result to a variable so it's easy to reuse it since you make the call more than once.